src/repository/WebhookFileRepository.js

```diff
@@ -32,11 +32,33 @@ class WebhookFileRepository {
   deleteById = async (id: string): Promise<void> =>
     this._fileManager.deleteFile(`${id}.json`);
 
-  getAll = async (): Promise<Array<Webhook>> =>
-    this._fileManager.getAllData();
+  getAll = async (userID: ?string = null): Promise<Array<Webhook>> => {
+    const allData = this._fileManager.getAllData();
 
-  getById = async (id: string): Promise<?Webhook> =>
-    this._fileManager.getFile(`${id}.json`);
+    if (userID) {
+      return Promise.resolve(
+        allData.filter(
+          (webhook: Webhook): boolean =>
+          webhook.ownerID === userID,
```

Comment:
this should be tabbed in.

When these functions span 2 lines, it should look like:
```
(webhook: Webhook): boolean =>
  wehbook.ownerID === userID,
```